### PR TITLE
React: Hookup unlinked files with react-query

### PR DESCRIPTION
**Pull request by frewmack**
Part of #284

### CODE REVIEW

File: react/src/hooks/unlinked/useUnlinkedFilesQuery.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Return result of GET /api/unlinked.
+ *
+ * That is, return a sorted list of filenames
+ * for all unlinked files in MinIO.
+ */
+export function useUnlinkedFilesQuery() {
+    const result = useQuery<string[], Response>("unlinked", fetchFiles);
+    if (result.isSuccess)
+        result.data = result.data.sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()));
```
Comment (kevinlul) [inline react/src/hooks/unlinked/useUnlinkedFilesQuery.tsx:17]: Should we sort on the backend instead and the frontend is only responsible for keeping its own representation sorted?

**Review by kevinlul — APPROVED** [review]
LGTM!

File: react/src/hooks/unlinked/useUnlinkedFilesQuery.tsx (hunk 1 above)
Comment (frewmack) [reply react/src/hooks/unlinked/useUnlinkedFilesQuery.tsx:17]: Yeah that might be a good idea. I don't think there's any frontend actions that add to unlinked files, only remove, so no extra care needs to be taken to keep the list sorted.

Comment (kevinlul) [reply react/src/hooks/unlinked/useUnlinkedFilesQuery.tsx:17]: Some frontend actions do add to the list but care was already taken then to sort the list before the state is updated. You can just call `.sort()` or `sorted` on the list before it's passed to `jsonify` in the endpoint.

**Comment by kevinlul** [comment]
Merging #386 resulted in an nontrivial merge conflict here 👀

**Pull request merged**